### PR TITLE
More addon template tidying

### PR DESCRIPTION
**Pull request by lgarvey**

### CODE REVIEW

**Review by yusufsheiqh — APPROVED** [review]

File: templates/addons/env/addons.parameters.yml [hunk 1]
```diff
@@ -4,7 +4,5 @@
 # To be fixed under "DBTP-295 Only add the required parameters to addons.parameters.yml"
 Parameters:
   EnvironmentSecurityGroup: !Ref EnvironmentSecurityGroup
-  LBARN: !Ref PublicLoadBalancer
```
Comment (WillGibson) [inline templates/addons/env/addons.parameters.yml:7]: Isn't `LBARN` used in `templates/addons/env/waf.yml`?

Comment (WillGibson) [reply templates/addons/env/addons.parameters.yml:7]: Oh, but that's going away isn't it.

Comment (lgarvey) [reply templates/addons/env/addons.parameters.yml:7]: Yes, it's dead in the water.  Jayesh is building the replacement

**Review by WillGibson — APPROVED** [review]

File: templates/addons/env/addons.parameters.yml [hunk 2]
```diff
@@ -4,7 +4,5 @@
...
-  PrivateSubnet1: !Ref PrivateSubnet1
-  PrivateSubnet2: !Ref PrivateSubnet2
-  VPCID: !Ref VPC
+  PrivateSubnet: !Join [ ',', [ !Ref PrivateSubnet1, !Ref PrivateSubnet2, ] ]
```
Comment (WillGibson) [inline templates/addons/env/addons.parameters.yml:7]:
```suggestion
  PrivateSubnets: !Join [ ',', [ !Ref PrivateSubnet1, !Ref PrivateSubnet2, ] ]
```

**Pull request merged**